app/assets/stylesheets/common/base/sidebar-more-section-links.scss

```diff
@@ -0,0 +1,32 @@
+.sidebar-more-section-links-details {
+  margin-left: 1.5em;
+
+  .sidebar-more-section-links-details-summary {
```

Comment:
I think we are supposed to be following BEM now? Not 100% sure if we are only doing it in chat or everywhere yet, if not just don't worry. Under BEM this would be `&__summary` so the final class would be `sidebar-more-section-links-details__summary`

Reply:
I think only chat is following it at the moment and I think the rules are not even finalised yet.

Reply:
Okay no problem, just thought I would check.